www/js/model.js

```diff
@@ -259,14 +276,40 @@ define([
 			},
 			setColor: function(/*string*/ id, /*string*/ color){
 				this.getNode(id).color = color;
-			}
+			},
+			setAuthorStatus: function(/*string*/ id, /*string*/ part, /*string*/ status){
+				// Summary: function to set the status of node editor in author mode.
+				if(!this.getNode(id).authorStatus){
```

Comment:
No it is not genus, it is  a separate field authorStatus for a node ( applied in applyDirectives each time) . Please check applyDirectives in controller.js and updateModelStatus in con-author js